dotcom-rendering/src/components/ProductCardButtons.tsx

```diff
@@ -0,0 +1,35 @@
+import type { ProductCta } from '../types/content';
+import { ProductLinkButton } from './ProductLinkButton';
+
+const getLabel = (cta: ProductCta): string => {
+	const overrideLabel = cta.text.trim().length > 0;
+	return overrideLabel ? cta.text : `${cta.price} at ${cta.retailer}`;
+};
+
+export const ProductCardButtons = ({
+	productCtas,
+	dataComponent,
+}: {
+	productCtas: ProductCta[];
+	dataComponent?: string;
+}) => {
+	return (
```

Comment:
Small thing, but as this is the only statement in this function, you could remove the `return`, allow the arrow function to implicitly return the single expression, and save yourself a level of indentation?